packages/runner/src/runner.ts

```diff
@@ -203,8 +277,10 @@ export class Runner<X extends AppConfig> implements IComponent {
     }
 
     async hookupFifoStreams() {
+        // @TODO bring it back to normal
```

Comment:
Nah, that can actually not be such a bad idea.

Reply:
Leave it as you made it, logger will be initialized ahead of other streams so if anything goes wrong there "a chance" log will show us something.

Reply:
fixed